package-lock.json

```diff
@@ -0,0 +1,338 @@
+{
+  "name": "valuecollection",
+  "version": "1.0.1",
```

Comment:
I'm cool with adding a package-lock.json file. The version field here doesn't match the one in package.json though and I think it should be "1.0.2". I think you need to "npm install" again or something. I'm also happy to take a patch with no package-lock.json file.